docs/frc/1-Basics/9-Drivetrain.md

```diff
@@ -72,6 +72,8 @@ We'll declare our state(s) in the form of an [enum:](https://frc1257.github.io/r
 
 We then add a private variable that'll hold the default state of the subsystem, which, in this case, will be `MANUAL`. Having a separate default state variable is good for expandability in the program and also reduces confusion during robot testing or competition. Finally, we initialize our subsystem-wide state variable with the default state.
 
+The state we introduced above can be considered the "operating mode" or "configuration" the subsystem is in. For our purposes here, that would be manual control, where the driver controls the motor speeds. Now, take a step back to the `speedForward` and `speedTurn` variables. You can think of these as an extra part or extension of the state that we are keeping track of, as they represent more information *about* the state and current operating behavior.
```

Comment:
I think rather than saying "take a step back," you should just put the `speedForward` and `speedTurn` variables in this section with the state.